### ISSUE TITLE
False accusations of the last module standing

### ISSUE THREAD

**Issue by BadIdeasBureau** [Moerill/fvtt-find-the-culprit#2]
If Find the Culprit is left with only one module remaining, it will accuse that module of being the culprit.  However, if the problem is a 3-way conflict (or more complicated than that), then this might not be the case.  It should do a check with just the "required" modules and the "culprit" to confirm the result.

**Comment by Faeranne** [comment]
Agreed.  I am trying to track down an issue with theme.min.js erroring out when I open a text editor.  Last Module has accused 3 different modules at this point.

**Comment by MrVauxs** [comment]
Same here. On large 150+ module worlds, tracking down the culprit starts to become impossible.
